.github/workflows/test.yml

```diff
@@ -17,16 +17,15 @@ jobs:
       run: |
         apt-get -qq update
         apt-get -qq upgrade -y
-        apt-get -qq install -y curl libasio-dev libtinyxml2-dev
-    # TODO(mikaelarguedas) switch back to ros-tooling/action-ros-ci once
-    # https://github.com/ros-tooling/action-ros-ci/pull/109 is released
-    - uses: mikaelarguedas/action-ros-ci@sros2-version
+        apt-get -qq install -y curl libasio-dev libssl-dev libtinyxml2-dev
+    - uses: ros-tooling/action-ros-ci@0.1.0
```

Comment:
Just an fyi - 0.1.1 released between this PR opening and now, minor changes mostly related to older distros

Reply:
Would be nice if `ros-tooling/action-ros-ci` had a rolling tag like e.g. `actions/setup-node@v1`.

Reply:
there is now a v0.1 tag for action-ros-ci.
Switched to using that in https://github.com/ros2/sros2/pull/247/commits/52fa3955749ba926628c9b49f75f6638964630f8

@kyrofa in the past I believe you were in favor of pinned versions rather than rolling ones. How do you feel about this change ?

Reply:
If it's versioned semantically, I'm cool with it.